demucs/apply.py

```diff
@@ -129,13 +130,22 @@ def tensor_chunk(tensor_or_chunk):
         return TensorChunk(tensor_or_chunk)
 
 
+def _replace_dict(_dict: tp.Optional[dict], *subs: tp.Tuple[tp.Hashable, tp.Any]) -> dict:
```

Comment:
this function is doing in place editing, I think this is a bit dangerous ! you should make a copy of the dict :)